packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1372,11 +1371,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 return { ...attemptData, reason: "disconnected" };
             }
 
-            if (this.runtimeOptions.runGC) {
+            if (!this.runtimeOptions.disableGC) {
                 // Get the container's GC data and run GC on the reference graph in it.
                 const gcData = await this.dataStores.getGCData();
                 const { referencedNodeIds } = runGarbageCollection(gcData.gcNodes, [ "/" ], this.logger);
 
+                // Update our summarizer node's used routes. Updating used routes in summarizer node before summarizing
+                // is required and asserted by the the summarizer node.
+                this.summarizerNode.updateUsedRoutes([""]);
```

Comment:
Just to clarify, this is [""] because the root is always referenced?

Reply:
Yes, that's correct.

Reply:
I will add this to the comment there.